docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst

```diff
@@ -0,0 +1,296 @@
+
+             ov::genai::WhisperPipeline pipeline{model_path};
+
+             ov::genai::WhisperGenerationConfig config{model_path + "/generation_config.json"};
+             config.max_new_tokens = 100;
+             // 'task' and 'language' parameters are supported for multilingual models only
+             config.language = "<|en|>";
+             config.task = "transcribe";
+             config.return_timestamps = true;
+
+             auto streamer = [](std::string word) {
+                 std::cout << word;
+                 return false;
+             };
+
+             auto result = pipeline.generate(raw_speech, config, streamer);
+
+             std::cout << "\n";
+
+             for (auto& chunk : *result.chunks) {
+                 std::cout << "timestamps: [" << chunk.start_ts << ", " << chunk.end_ts << "] text: " << chunk.text << "\n";
+             }
+         } catch (const std::exception& error) {
+             try {
+                 std::cerr << error.what() << '\n';
+             } catch (const std::ios_base::failure&) {
+             }
+             return EXIT_FAILURE;
+         } catch (...) {
+             try {
+                 std::cerr << "Non-exception object thrown\n";
+             } catch (const std::ios_base::failure&) {
+             }
+             return EXIT_FAILURE;
+         }
+
+
+      For more information, refer to the
+      `C++ sample <https://github.com/openvinotoolkit/openvino.genai/blob/master/samples/cpp/whisper_speech_recognition/README.md>`__.
+
+
+Using GenAI in Chat Scenario
+############################
+
+For chat scenarios where inputs and outputs represent a conversation, maintaining KVCache across inputs
+may prove beneficial. The ``start_chat`` and ``finish_chat`` chat-specific methods are used to
+mark a conversation session, as shown in the samples below:
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+      .. code-block:: python
+
+         import openvino_genai as ov_genai
+         pipe = ov_genai.LLMPipeline(model_path)
+
+         pipe.set_generation_config({'max_new_tokens': 100)
+
+         pipe.start_chat()
+         while True:
+            print('question:')
+            prompt = input()
+            if prompt == 'Stop!':
+               break
+            print(pipe.generate(prompt))
+         pipe.finish_chat()
+
+
+      For more information, refer to the
+      `Python sample <https://github.com/openvinotoolkit/openvino.genai/blob/master/samples/python/chat_sample/README.md>`__.
+
+   .. tab-item:: C++
+      :sync: cpp
+
+      .. code-block:: cpp
+
+         int main(int argc, char* argv[]) {
+            std::string prompt;
+
+            std::string model_path = argv[1];
+            ov::genai::LLMPipeline pipe(model_path, "CPU");
+
+            ov::genai::GenerationConfig config = pipe.get_generation_config();
+            config.max_new_tokens = 100;
+            pipe.set_generation_config(config)
```

Comment:
is there a ; missing?

Reply:
I suppose it is. The code moved from the main article seems to be outdated and I believe it should be changed to https://github.com/openvinotoolkit/openvino.genai/blob/master/samples/cpp/chat_sample/chat_sample.cpp#L6C1-L33C2

@andrei-kochin can you confirm?

Reply:
@sgolebiewski-intel yes, let's align with the sample

Reply:
I have aligned it:
https://github.com/openvinotoolkit/openvino/pull/27062/commits/08e4a066c128c5e0ce4daca9d6e14488132070ba